src/main/java/com/staff_management_system/com/database_connection_package/HourlyEmployeeDataAccess.java

```diff
@@ -0,0 +1,266 @@
+package com.database_connection_package;
+
+import com.exception_package.*;
+import com.staff_classes_package.HourlyEmployee;
+
+import java.sql.*;
+
+/**
+ * Subclass HourlyEmployeeDataAccess that holds default and parameterized constructors with values.
+ *
+ * @author Nour Al Jarrah
+ * @version 2.0
+ * @since 1.0
+ */
+public class HourlyEmployeeDataAccess {
+
+    /**
+     * variables for all HourlyEmployee instance attribute values
+     */
+    long id;
+    String password;
+    String firstName;
+    String lastName;
+    String role;
+    Date startDate;
+    String sin;
+    long phoneNumber;
+    String emailAddress;
+    String address;
+    float hourly;
+    boolean enabled;
+    String department;
+    // retrieve employee data
+    /**
+     * variables for the database connection
+     */
+    private HourlyEmployee anHourlyEmployee = null;
+    private Connection aConnection;
+    private Statement aStatement;
+
+    /**
+     * @param c establish the database connection
+     */
+
+    public void initialize(Connection c) {
+        try {
+            aConnection = c;
+            aStatement = aConnection.createStatement();
+        } catch (SQLException e) {
+            e.printStackTrace();
+        }
+    }
+
+    /**
+     * close the database connection
+     */
+    public void terminate() {
+        try {
+            aStatement.close();
+        } catch (SQLException e) {
+            e.printStackTrace();
+        }
+    }
+
+
+    /**
+     * retrieve the database connection
```

Comment:
the comment is not accurate anymore. Remember to keep your comment upto date as you change your code. It is very easy for code and comment to go out of sync.

Reply:
I will update all the comments in the code.